src/libstd/sync/condvar.rs

```diff
@@ -18,6 +18,19 @@ use sys_common::poison::{self, LockResult};
 use sys::time::SteadyTime;
 use time::Duration;
 
+/// A type indicating whether a timed wait on a condition variable returned
+/// due to a time out or not.
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+#[unstable(feature = "wait_timeout", reason = "newly added")]
```

Comment:
This'll need an explicit `issue` tag, and could you also add the unstable tag to the `timed_out` method?